doc/classes/EditorFileSystem.xml

```diff
@@ -54,11 +54,28 @@
 				Check if the source of any imported resource changed.
 			</description>
 		</method>
+		<method name="reimport_files">
+			<return type="void" />
+			<argument index="0" name="files" type="PackedStringArray" />
+			<description>
+				Reimports a set of files.
+				Call this if these files or their [code].import[/code] files were directly edited by script or an external program.
+
+				If the file type changed or the file was newly created, use [method update_file] or [method scan].
+
+				Note: This function blocks until the import is finished. However, the main loop iteration,
+				including timers and [code]_process[/code], will occur during the import process due to progress bar updates.
+				Avoid calls to reimport_files or scan while an import is in progress.
```

Comment:
In the doc XML, 1 line == 1 paragraph. So the empty lines and line breaks will add unnecessary paragraphs in the in-editor help and the generate HTML.